src/Components/Task.js

```diff
@@ -0,0 +1,60 @@
+import React, { useEffect, useState } from 'react';
+
+const Task = ({ description, setTaskList, tasks, id, setTaskToBeEdited, setNewDescription }) => {
+    const [isStrikeThrough, setStrikeThrough] = useState(false);
+
+    const removeTask = (id) => {
+        return setTaskList(tasks.filter(task => task.id !== id))
```

Comment:
No need to return here as the state is being updated so it will automatically be reflected when the component re-renders.